content/posts/0024-bionic-builder-unsafe.md

```diff
@@ -0,0 +1,90 @@
+
+With `pack`, users should run `pack config default-builder`. It should say that they are using a Jammy builder.
+
+```
+> pack config default-builder
+The current default builder is paketobuildpacks/builder-jammy-base
+```
+
+If it does not, then run `pack config default-builder paketobuildpacks/builder-jammy-base` (or `-full` or `-tiny`, based on your needs).
+
+### Check your CI pipelines
+
+Users should also check their CI pipelines, build scripts, and other automations for the usage of the Bionic Builders. 
+
+- Users may call `pack build` with the `-B` or `--builder` flag to set a builder. 
+- Users may include a `project.toml` file which might have a `builder = <builder>` property set in the `[build]` block to set a builder.
+- It is less likely, but users can also call `pack config default-builder <builder>` to set a builder in CI.
+
+Users may also want to check for the names of the older builders in your CI/pipelines:
+
+- `paketobuildpacks/builder:tiny`
+- `paketobuildpacks/builder:base`
+- `paketobuildpacks/builder:full`
+- `paketobuildpacks/builder:buildpackless-tiny`
+- `paketobuildpacks/builder:buildpackless-base`
+- `paketobuildpacks/builder:buildpackless-full`
+
+Any usage of these builders should be replaced with the equivalent Jammy builder.
+
+- `paketobuildpacks/builder-jammy-tiny:latest`
+- `paketobuildpacks/builder-jammy-base:latest`
+- `paketobuildpacks/builder-jammy-full:latest`
+- `paketobuildpacks/builder-jammy-buildpackless-tiny:latest`
+- `paketobuildpacks/builder-jammy-buildpackless-base:latest`
+- `paketobuildpacks/builder-jammy-buildpackless-full:latest`
+
+### Upgrade your Spring Boot Apps
+
+[Spring Boot Users will continue to default to the Bionic Stack until Spring Boot 3.2](https://spring.io/blog/2023/09/22/paketo-buildpacks-bionic-end-of-support). Please read the Spring Blog post and perform the actions listed there. That will upgrade your app to use the Jammy stack.
+
+Please also consider upgrading to Spring Boot 3.2 when it is available. Spring Boot 3.2+ will default to using the Jammy builders.
+
+### Help! I need Bionic
+
+We hope that everyone will be able to upgrade, but we recognize that there may be cases where a user cannot upgrade or cannot upgrade immediately.
+
+For these users:
+
+1. Your builds will break during the brownout. Don't panic! They will start working again when the brownout is over. Grab your towel or a [Pan Galactic Gargle Blaster](https://en.wikipedia.org/wiki/Zaphod_Beeblebrox) and wait it out.
+2. When the brownout is over, you will need to switch the image tag on your builder to the following:
+
+   - `paketobuildpacks/builder:tiny-unsafe`
+   - `paketobuildpacks/builder:base-unsafe`
+   - `paketobuildpacks/builder:full-unsafe`
+   - `paketobuildpacks/builder:buildpackless-tiny-unsafe`
+   - `paketobuildpacks/builder:buildpackless-base-unsafe`
+   - `paketobuildpacks/builder:buildpackless-full-unsafe`
```

Comment:
I think we should rather introduce the unsafe tags already. I.e. whoever needs Bionic can already fix the CI, no need to potentially fail again with the permanent removal. WDYT?

Reply:
Yes, that sounds good. I can add that right after I remove the other tags.